src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorFrontendEndpointObject.cs

```diff
@@ -57,47 +52,53 @@ public class NewAzureRmFrontDoorFrontendEndpointObject : AzureFrontDoorCmdletBas
         public int SessionAffinityTtlInSeconds { get; set; }
 
         /// <summary>
-        /// The resource id of Web Application Firewall policy for each host (if applicable)
+        /// The resource id of Web Application Firewall policy for each host (if applicable).
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "The resource id of Web Application Firewall policy for each host (if applicable)")]
         public string WebApplicationFirewallPolicyLink { get; set; }
 
         /// <summary>
-        /// The source of the SSL certificate
+        /// The source of the SSL certificate. Part of CustomHttpsConfiguration.
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "The source of the SSL certificate")]
         [PSArgumentCompleter("AzureKeyVault", "FrontDoor")]
         public string CertificateSource { get; set; }
 
         /// <summary>
-        /// Defines the TLS extension protocol that is used for secure delivery
+        /// The minimum TLS version required from the clients to establish an SSL handshake with Front Door. Part of CustomHttpsConfiguration.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The minimum TLS version required from the clients to establish an SSL handshake with Front Door.")]
+        public string MinimumTlsVersion { get; set; }
+
+        /// <summary>
+        /// Defines the TLS extension protocol that is used for secure delivery. Part of CustomHttpsConfiguration.
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "The TLS extension protocol that is used for secure delivery")]
         [PSArgumentCompleter("ServerNameIndication", "IPBased")]
         public string ProtocolType { get; set; }
 
         /// <summary>
-        /// Defines the TLS extension protocol that is used for secure delivery
+        /// Defines the TLS extension protocol that is used for secure delivery. Part of CustomHttpsConfiguration.
```

Comment:
>Defines the TLS extension protocol that is used for secure delivery [](start = 12, length = 67)

incorrect comment

Reply:
Removed "Part of CustomHttpsConfiguration" for all parameters in CustomHttpsConfiguration.